oaknational/Oak-Web-Application
# fix: increase character count limit for teacher tips [LESQ-1105]

## Description

Music year: 2010

- increase character count limit for teacher tips on lesson page to match CAT restrictions

## Issue(s)

Fixes #
Teacher tips not showing on some lessons

## How to test

1. Go to https://deploy-preview-2884--oak-web-application.netlify.thenational.academy/teachers/programmes/biology-secondary-ks4-higher-aqa/units/coordination-and-control-the-human-nervous-system/lessons/human-reaction-time-practical-u5542#lesson-details
3. You should see the tooltip

## Screenshots

How it used to look (delete if n/a):

<img width="700" alt="Screenshot 2024-10-14 at 14 40 04" src="https://github.com/user-attachments/assets/d749e1c7-fb09-4fbd-aa90-573db1f8cdcb">


How it should now look:

<img width="700" alt="Screenshot 2024-10-14 at 14 40 17" src="https://github.com/user-attachments/assets/de1e2ee0-4336-49cc-8586-879657c56467">


## Checklist

Some more lessons with long teacher tips (300 characters):
- solve-problems-by-adding-and-subtracting-tens-and-ones
- choose-the-most-efficient-strategy-to-subtract-from-a-3-digit-number
- cardiff-as-a-capital-city
- dispersion-of-light-and-electromagnetic-radiation
- convert-between-times-given-in-12-and-24-hours
- sound-waves
